packages/server/lib/browsers/browser-cri-client.ts

```diff
@@ -105,19 +111,20 @@ export class BrowserCriClient {
    * @returns the chrome remote interface wrapper for the target
    */
   attachToTargetUrl = async (url: string): Promise<CRIWrapper.Client> => {
-    debug('Attaching to target url %s', url)
-    const { targetInfos: targets } = await this.browserClient.send('Target.getTargets')
+    return retryWithIncreasingDelay(async () => {
+      debug('Attaching to target url %s', url)
+      const { targetInfos: targets } = await this.browserClient.send('Target.getTargets')
 
-    const target = targets.find((target) => target.url === url)
+      const target = targets.find((target) => target.url === url)
 
-    // TODO: Move this into packages/error post merge: https://github.com/cypress-io/cypress/pull/20072
```

Comment:
Is this the TODO worth persisting/doing?

Reply:
This fix actually negates the rationale for the TODO (which is nice). This scenarios is now handled by this catch block:

https://github.com/cypress-io/cypress/pull/21055/files#diff-b5b66d85423fa714add2e866448d3411b0f1cdf2cb4b4a95aa80394fa3edd1b3R57

which properly displays the appropriate error.